src/components/Table/UncontrolledTable.js

```diff
@@ -78,7 +78,10 @@ export default class UncontrolledTable extends React.Component {
   };
 
   get allSelected() {
-    return this.props.rows.length && this.state.selected.length === this.props.rows.length;
+    const selectableRows = this.props.rows.filter((row) =>
+      Object.hasOwn(row, 'disabled') ? !row.disabled : true
```

Comment:
Same here, if `row.disabled` is not set, it is `undefined`. `!undefined` is `true` so the`Object.hasOwn` seems like is isn't needed.